app/controllers/vm_common.rb

```diff
@@ -1254,7 +1254,10 @@ def replace_right_cell(options = {})
     ])
 
     presenter[:clear_gtl_list_grid] = @gtl_type && @gtl_type != 'list'
-    presenter[:clear_tree_cookies] = "edit_treeOpenStatex" if @sb[:action] == "policy_sim"
+    if @sb[:action] == "policy_sim"
+      presenter[:clear_tree_cookies] = "edit_treeOpenStatex"
+      presenter[:tree_expandAll] = 0
```

Comment:
Why a number, why not boolean?

Reply:
Great question, changing.  For some reason my mind went to 0 and 1.